gtsfm/evaluation/metrics_report.py

```diff
@@ -191,13 +285,16 @@ def generate_metrics_report_html(metrics_groups: List[GtsfmMetricsGroup], html_p
         f.write(get_html_header())
 
         # Iterate over all metrics groups
-        for metrics_group in metrics_groups:
+        for i, metrics_group in enumerate(metrics_groups):
 
             # Write name of the metric group in human readable form.
             f.write(get_html_metric_heading(metrics_group.name))
 
             # Write plots and tables.
-            table, plots_fig = get_figures_for_metrics(metrics_group)
+            if colmap_files_dirpath == "":
```

Comment:
this is probably not a great check, it might be better to have a None default, and check for it instead.

Reply:
Done.